Units/parser-gdscript.r/sample/input.gd

```diff
@@ -0,0 +1,77 @@
+# A file is a class!
```

Comment:
Could you add the origin of the test input?
```
# Taken from https://docs.godotengine.org/en/latest/tutorials/scripting/gdscript/gdscript_basics.html
```
or
```
# Derrived from https://docs.godotengine.org/en/latest/tutorials/scripting/gdscript/gdscript_basics.html
```

Reply:
Went with taken from, I did change one line slightly so that it would test typed constants but it's still basically taken from the docs.